plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/TableViewerBuilder.java

```diff
@@ -50,13 +51,14 @@ public TableViewerBuilder(TableViewer viewer, Composite tableContainer) {
 		viewer.setComparer(new EqualityComparer());
 		this.tableLayout = new TableColumnLayout();
 		tableContainer.setLayout(tableLayout);
-		FontData fd = viewer.getTable().getFont().getFontData()[0];
+		normalFont = viewer.getTable().getFont();
+		FontData fd = normalFont.getFontData()[0];
 		fd.setStyle(SWT.ITALIC);
-		italic = new Font(null, fd);
+		italicFont = new Font(null, fd);
```

Comment:
There is also JFaceResources.DIALOG_FONT that may be more relevant as we have a dialog in this case.
I checked that in my environment JFaceResources.getDefaultFont(), JFaceResources.getDialogFont() and Table.getFont() return fonts with the same name, height and style, but that can be different in other environments.

And, for an experiment I made a call JFaceResources.getDialogFont().dispose(). Wow, it broke all dialogs Eclipse!